frontend/constants/networks.ts

```diff
@@ -0,0 +1,16 @@
+export const NETWORKS = {
```

Comment:
realize this slightly differs from our current "constants" setup, but im just getting processes started for testnet vs mainnet distinction and tis is a better structure. future PR dealing with it should be responsible for full clean up rather than doing it in this PR